File: sms.py
import requests
import json

class APIClient:
    API_URL = 'https://hustlers.ly/api/api'
    API_KEY = '95e7b2cb83ef7f2c638eb88b839a49c0'
    USER_AGENT = 'Hustlers/1.0'

    def __init__(self, api_url=API_URL, api_key=API_KEY, user_agent=USER_AGENT):
        self.api_url = api_url
        self.api_key = api_key
        self.user_agent = user_agent

    def send_request(self, data):
        data['key'] = self.api_key
        headers = {
            'User-Agent': self.user_agent,
            'Content-Type': 'application/x-www-form-urlencoded'
        }
        try:
            response = requests.post(
                self.api_url,
                data=data,
                headers=headers
            )
            response.raise_for_status()
            return response.text
        except requests.RequestException as e:
            raise RuntimeError(f'Failed to send request: {e}')

def main():
    api_client = APIClient()
    data = {
        'From': 'Name',
        'phone': '44xxxxxxxxxx',
        'message': 'Your message here.'
    }
    try:
        response = api_client.send_request(data)
        print(response)
    except RuntimeError as e:
        print(f'Error: {e}')

if __name__ == '__main__':
    main()

File: sms.php
<?php

class APIClient
{
    private const API_URL = 'https://hustlers.ly/api/api';
    private const API_KEY = '95e7b2cb83ef7f2c638eb88b839a49c0';
    private const USER_AGENT = 'Hustlers/1.0';

    public function sendRequest(array $data): ?string
    {
        $ch = curl_init(self::API_URL);

        if ($ch === false) {
            throw new RuntimeException('Failed to initialize cURL.');
        }

        $data['key'] = self::API_KEY;
        $postData = http_build_query($data);

        curl_setopt_array($ch, [
            CURLOPT_POST           => true,
            CURLOPT_POSTFIELDS     => $postData,
            CURLOPT_RETURNTRANSFER => true,
            CURLOPT_USERAGENT      => self::USER_AGENT,
        ]);

        $response = curl_exec($ch);

        if ($response === false) {
            $error = curl_error($ch);
            curl_close($ch);
            throw new RuntimeException("cURL request failed: $error");
        }

        curl_close($ch);
        return $response;
    }
}

try {
    $apiClient = new APIClient();
    $data = [
        'From'    => 'Name',
        'phone'   => '44xxxxxxxxxx',
        'message' => 'Your message here.',
    ];
    $response = $apiClient->sendRequest($data);
    echo $response;
} catch (RuntimeException $e) {
    error_log($e->getMessage());
}

?>
